### DISCUSSION TITLE
Why did I add my own custom css in "_variable-custom.scss" and it doesn't work?

### DISCUSSION THREAD

**Discussion by heihahehe** [thuliteio/doks#1158] [Q&A]
Why did I add my own custom css in "_variable-custom.scss" and it doesn't work?
![0240102114831](https://github.com/gethyas/doks/assets/152707762/49b88ec2-6c68-48d0-90d7-c2a22da233bd)

**Comment by h-enk** [comment]
Because of the import order — see https://github.com/gethyas/doks-core/blob/main/assets/scss/app.scss

- Use `/assets/scss/common/_variables-custom.scss` for your custom (S)CSS variables
- Use `/assets/scss/common/_custom.scss` for your custom (S)CSS code
